--- a/Remoting/Core/vtkPVFileInformation.cxx-org	2019-09-26 13:59:18.000000000 -0700
+++ b/Remoting/Core/vtkPVFileInformation.cxx	2019-11-02 17:42:04.185351113 -0700
@@ -1201,6 +1201,7 @@
 //-----------------------------------------------------------------------------
 std::string vtkPVFileInformation::GetParaViewSharedResourcesDirectory()
 {
+#if 0
   // Look for where the function "GetVTKVersion." lives.
   auto vtk_libs = vtkGetLibraryPathForSymbol(GetVTKVersion);
 
@@ -1222,6 +1223,8 @@
   }
 
   return resource_dir;
+#endif
+  return "/usr/share/ParaView-" PARAVIEW_VERSION;
 }
 
 //-----------------------------------------------------------------------------
